### ISSUE TITLE
Action shape가 의도한대로 return이 되지 않는다.

### ISSUE THREAD

**Issue by grooviiee** [grooviiee/python_rl#28]
Bool을 했는데 왜 float 형이 출력될까???

**Comment by grooviiee** [comment]
sample을 했을 때는 값이 제대로 나오는데, nn을 돌렸을 때 return되는 값이 아래처럼 이상하다.
조금 더 살펴볼 것
[ACTLayer_forward] (MBS) action_logits: FixedCategorical(probs: tensor([[1.]]), logits: tensor([[0.]])), actions: torch.Size([1, 1])
